### ISSUE TITLE
Reference to csproj project

### ISSUE THREAD

**Issue by tangkhaiphuong** [dotnet-script/dotnet-script#691]
Does script support reference to csproj instead of dll?
For example
`#r “../library/util.csproj”`

**Comment by filipw** [comment]
No, this is not supported (and I don't think it will ever be)

**Comment by diegosiao** [comment]
What about another file @filipw?

**Comment by filipw** [comment]
@diegosiao yes you can reference another csx using `#load "...path to .csx"`
